### PR TITLE
Merge `development` into `main`

### PR DESCRIPTION
**Pull request by germartinez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 description: Everything you need to know to start building on Safe.
 ---
 
-# 👋 Introduction
+# Introduction
 
-Safe makes it easy for anyone to make and use accounts on the blockchain. These docs explain how to do this.
+Safe brings digital ownership of accounts to everyone by building universal and open contract standards for the custody of digital assets, data, and identity with the Safe ecosystem. These docs explain how to build with Safe.
 
 ### How to Read this Guide
```
Comment (yagopv) [inline README.md]: Remember to remove/change

File: reference/safe-core-sdk/auth-kit/README.md [hunk 2]
```diff
@@ -1,133 +1,32 @@
...
 
-To add the Auth kit to your project, run:
+To add the Auth Kit to your project, run:
 
 ```bash
 yarn add @safe-global/auth-kit
 ```
 
-We currently offer several providers in the form of "packs". A pack is a piece of code that works with the chosen provider to give Safe users access to new services.
+The Aut Kit can integrate several providers in the form of **"packs"**. A pack is a piece of code that works with the chosen provider to give Safe users access to new services.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/README.md:13]:
```suggestion
The Auth Kit can integrate several providers in the form of **"packs"**. A pack is a piece of code that works with the chosen provider to give Safe users access to new services.
```

File: README.md [hunk 3]
```diff
@@ -22,27 +22,33 @@ We recommend starting with Learn and seeing some tutorials there to learn more a
...
 
-<figure><img src=".gitbook/assets/download.png" alt=""><figcaption><p><a href="https://viewer.diagrams.net/?tags=%7B%7D&#x26;target=blank&#x26;highlight=0000ff&#x26;edit=_blank&#x26;layers=1&#x26;nav=1&#x26;page-id=atRejJyS5DeNAtDboIeV&#x26;title=Safe%20Diagrams.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1WcTgdHoQttJ0K_fV8mDg-RmDZRYGe3D-%26export%3Ddownload">View Diagram</a></p></figcaption></figure>
+<figure><img src=".gitbook/assets/diagram-safe-ecosystem.png" alt=""><figcaption><p><a href="https://viewer.diagrams.net/index.html?tags=%7B%7D&target=blank&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=atRejJyS5DeNAtDboIeV&title=Safe%20Diagrams.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1WcTgdHoQttJ0K_fV8mDg-RmDZRYGe3D-%26export%3Ddownload">View Diagram</a></p></figcaption></figure>
 
 ### Contact Us
 
-1. [Tweet us on Twitter @safe](https://twitter.com/safe)
-2. [Ask 'gnosis-safe' Questions on the Ethereum Stack Exchange](https://ethereum.stackexchange.com/questions/tagged/gnosis-safe)
```
[the remark this answers is not in the document]
Comment (germartinez) [reply README.md:43]: It should not be removed, but I remember updating the tag to `safe-core`. Probably it was in a different place, this must be updated as well ;)

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
+# Web3AuthModalPack
+
+Web3Auth is a pluggable authentication infrastructure for Web3 wallets and applications. It simplifies onboarding for both mainstream and crypto native users, offering experiences tailored to their preferences. It supports all social logins, web and mobile native platforms, wallets, and other key management methods, creating a cryptographic key provider specific to the user and application.
+
+The `Web3AuthModalPack` enables the use of [Web3Auth modal SDK](https://web3auth.io/docs/sdk/web/modal/) with added Safe capabilities, such as retrieving the related Safe addresses for a given externally-owned account created using Web3Auth services.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:5]:
```suggestion
The `Web3AuthModalPack` enables the use of [Web3Auth modal SDK](https://web3auth.io/docs/sdk/pnp/web/modal) with added Safe capabilities, such as retrieving the related Safe addresses for a given externally-owned account created using Web3Auth services.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+yarn add @safe-global/auth-kit @web3auth/base @web3auth/modal
+```
+
+Choose the adapters to use with Web3Auth, and add them. For example, to use the [OpenLogin](https://docs.openlogin.com/) adapter, you must install the following dependency:
+
+```bash
+yarn add @web3auth/openlogin-adapter
+```
+
+Refer to the [supported adapters](https://web3auth.io/docs/sdk/web/adapters/) in the official documentation.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:21]:
```suggestion
Refer to the [supported adapters](https://web3auth.io/docs/sdk/pnp/web/adapters/#currently-available-wallet-adapters) in the official documentation.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+}
+```
+- `txServiceUrl` - The URL of the Safe transaction service. It is used to retrieve the Safe addresses for an externally-owned account created using Web3Auth services. It is required to use the `Web3AuthModalPack` with Safe.
+
+**Caveats**
+You should always call the `init()` method afterwards before interacting with the pack.
+
+### init(web3AuthModalOptions, adapters[]?, modalConfig?)
+
+The init method initialize the required Web3Auth modal SDK and Safe services. It [instantiates the `Web3Auth`](https://web3auth.io/docs/quick-start?product=Plug+and+Play&sdk=Plug+and+Play+Web+Modal+SDK&platform=React#3-initialize-web3auth-for-your-preferred-blockchain) class and [configure the adapters](https://web3auth.io/docs/sdk/web/no-modal/initialize#configureadapteradapter). It calls the [`initModal`](https://web3auth.io/docs/sdk/web/modal/whitelabel#initmodal) method internally.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:53]:
```suggestion
The init method initialize the required Web3Auth modal SDK and Safe services. It [instantiates the `Web3Auth`](https://web3auth.io/docs/quick-start?product=Plug+and+Play&sdk=Plug+and+Play+Web+Modal+SDK&platform=React#3-initialize-web3auth-for-your-preferred-blockchain) class and [configure the adapters](https://web3auth.io/docs/sdk/pnp/web/no-modal/initialize#configureadapteradapter). It calls the [`initModal`](https://web3auth.io/docs/sdk/pnp/web/modal/whitelabel#initmodal) method internally.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+It initializes as well the web3 provider we can retrieve afterwards using the `getProvider()` method.
+
+**Params**
+
+- `web3AuthModalOptions` - The options to create the `Web3Auth` [instance](https://web3auth.io/docs/sdk/web/modal/initialize#instantiating-web3auth). You can send a configuration object matching the [official one](https://web3auth.io/docs/sdk/web/modal/initialize#web3authoptions) as we passthrough the options to the `Web3Auth` instance when we instantiate it inside the pack.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:59]:
```suggestion
- `web3AuthModalOptions` - The options to create the `Web3Auth` [instance](https://web3auth.io/docs/sdk/pnp/web/modal/initialize#instantiating-web3auth). You can send a configuration object matching the [official one](https://web3auth.io/docs/sdk/pnp/web/modal/initialize#web3authoptions) as we passthrough the options to the `Web3Auth` instance when we instantiate it inside the pack.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+- `adapters` - The adapters for the Web3Auth modal sdk. You can use any of the [supported adapters](https://web3auth.io/docs/sdk/web/adapters/). This prop accepts an array of adapters and the `Web3AuthModalPack` will call the [`configureAdapter`](https://web3auth.io/docs/sdk/web/no-modal/initialize#configureadapteradapter) method for each one of them internally.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:60]:
```suggestion
- `adapters` - The adapters for the Web3Auth modal sdk. You can use any of the [supported adapters](https://web3auth.io/docs/sdk/pnp/web/adapters/). This prop accepts an array of adapters and the `Web3AuthModalPack` will call the [`configureAdapter`](https://web3auth.io/docs/sdk/pnp/web/no-modal/initialize#configureadapteradapter) method for each one of them internally.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+⚠️ Remember to add the corresponding package for each adapter (e.g `@web3auth/openlogin-adapter` for OpenLogin).
+
+- `modalConfig` - The configuration for the Web3Auth modal sdk. You can explore the options [here](https://web3auth.io/docs/sdk/web/modal/whitelabel#initmodal). This options are used with the `initModal` method internally.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:64]:
```suggestion
- `modalConfig` - The configuration for the Web3Auth modal sdk. You can explore the options [here](https://web3auth.io/docs/sdk/pnp/web/modal/whitelabel#initmodal). This options are used with the `initModal` method internally.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+**Caveats**
+
+- To get the Safe addresses, instantiate the authKit with the `txServiceUrl` property in the config object. Otherwise, only the eoa will be returned.
+- ⚠️ This method currently returns the Safe addresses which the EOA is the owner of. It does not create a Safe. We are investigating ways to enhance the Auth Kit and the associated flows. ⚠️
+
+### signOut()
+
+Call this method to sign out the user and clean the session.
+
+Internally it calls the [`logout()`](https://web3auth.io/docs/sdk/web/modal/usage#web3authlogout) method from Web3Auth and clears the web3 provider stored in the instance.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:92]:
```suggestion
Internally it calls the [`logout()`](https://web3auth.io/docs/sdk/pnp/web/modal/usage#web3authlogout) method from Web3Auth and clears the web3 provider stored in the instance.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+
+### getUserInfo()
+
+Using `getUserInfo()` you will receive the user information derived from the pack you are using. It varies depending on the provider.
+
+Internally it calls the [`getUserInfo()`](https://web3auth.io/docs/sdk/web/modal/usage#getuserinfo) method from Web3Auth.
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:98]:
```suggestion
Internally it calls the [`getUserInfo()`](https://web3auth.io/docs/sdk/pnp/web/modal/usage#getuserinfo) method from Web3Auth.
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+**Caveats**
+⚠️ Web3Auth with Social Logins returns a provider that allows you to sign transactions without any UI confirmations, unlike regular wallets such as Metamask ⚠️
+
+### subscribe(event, handler)
+
+Allow to subscribe to authentication state changes. The event depends on the pack you are using so read the chosen pack documentation.
+
+**Params**
+
+- `event` - The event you want to subscribe to. The events are defined [in the documentation](https://web3auth.io/docs/sdk/web/no-modal/initialize#subscribing-the-lifecycle-events).
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:119]:
```suggestion
- `event` - The event you want to subscribe to. The events are defined [in the documentation](https://web3auth.io/docs/sdk/pnp/web/no-modal/initialize#subscribing-the-lifecycle-events).
```

File: reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+
+### signIn()
+
+`signIn()` calls internally the [`connect()`](https://web3auth.io/docs/sdk/web/modal/usage#connect) method from Web3Auth. It obtains a web3 provider and store it internally in the instance. Finally, retrieves the associated Safe addresses for the externally-owned account created using Web3Auth services.
+
+**Returns**
+An object with the derived externally-owned account address and the associated safe addresses.
+
+```typescript
+SafeAuthSignInData {
```
Comment (yagopv) [inline reference/safe-core-sdk/auth-kit/Web3AuthModalPack.md:77]:
```suggestion
AuthKitSignInData {
```

File: reference/safe-core-sdk/onramp-kit/README.md [hunk 14]
```diff
@@ -1,114 +1,34 @@
...
-- `options?` - The options for the pack. For some packs they will be required, while others don't. Check the pack documentation for details.
-
-**Returns**
-The outcome of interacting with the pack varies. Refer to the pack's documentation for more information.
-
-#### `close()`
+After the initialization, use the methods in the pack to interact with the chosen provider.
 
-Clean up after interacting with the pack. This method is usually called when you are done using the kit, such as when you leave the page, or when you want to end the interaction with the pack.
+There are some common methods (`open()`, `clore()`, etc.) that can be called regardless of the package being used.
```
Comment (yagopv) [inline reference/safe-core-sdk/onramp-kit/README.md:32]:
```suggestion
There are some common methods (`open()`, `close()`, etc.) that can be called regardless of the package being used.
```

File: reference/safe-core-sdk/onramp-kit/OnRampKitBasePack.md [hunk 15]
```diff
@@ -0,0 +1,55 @@
...
+
+This class is used to create new packs. Any new pack should extend this class and implement the abstract methods. Extending from the `OnRampKitBasePack` class will give the subclass access to common Safe features.
+
+```typescript
+class MyPack extends OnRampKitBasePack {
+  // Implementation of the abstract methods
+}
+```
+
+## Abstract methods that OnRamp packs have to implement
```
Comment (yagopv) [inline reference/safe-core-sdk/onramp-kit/OnRampKitBasePack.md:23]:
```suggestion
## Abstract methods that OnRamp packs must implement
```

File: reference/safe-core-sdk/onramp-kit/StripePack.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+**Caveats**
+You should always call the `init()` method afterwards before interacting with the pack.
+
+### init()
+
+Loads the Stripe scripts using the public key provided in the class instantiation and initialize the Stripe SDK.
+
+### open(stripeOpenOptions)
+
+Opens the Stripe widget in the chosen slot (CSS selector) inside a webpage.
```
Comment (yagopv) [inline reference/safe-core-sdk/onramp-kit/StripePack.md:50]:
```suggestion
Opens the Stripe widget in the chosen DOM node (CSS selector) inside a webpage.
```

File: reference/TEMPLATE.md [hunk 17]
```diff
@@ -37,11 +37,11 @@
 - Caveat2: {Description}
```
Comment (yagopv) [inline reference/TEMPLATE.md:37]:
```suggestion
- Caveats: {Description}
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 18]
```diff
@@ -0,0 +1,196 @@
+# Integration with Web3Auth
+
+This guide demonstrate how to create an externally-owned account using your email or social media account. Once authenticated, you can sign transactions and interact with any Safe Smart Accounts you own.
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [Create a Web3Auth account](https://web3auth.io)
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:8]:
```suggestion
- [Create a Web3Auth account](https://dashboard.web3auth.io)
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 19]
```diff
@@ -0,0 +1,196 @@
...
+    // https://chainlist.org/
+    rpcTarget: 'https://rpc.ankr.com/eth_goerli'
+  },
+  uiConfig: {
+    theme: 'dark',
+    loginMethodsOrder: ['google', 'facebook']
+  }
+}
+
+// https://web3auth.io/docs/sdk/web/modal/initialize#configuring-adapters
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:44]:
```suggestion
// https://web3auth.io/docs/sdk/pnp/web/modal/initialize#configuring-adapters
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 20]
```diff
@@ -0,0 +1,196 @@
...
+
+Create an instance of the [Web3AuthModalPack](https://github.com/safe-global/safe-core-sdk/tree/main/packages/auth-kit/src/packs/web3auth/Web3AuthModalPack.ts) using the required `Web3AuthConfig` configuration object.
+
+```typescript
+import { Web3AuthModalPack, Web3AuthConfig } from '@safe-global/auth-kit'
+import { Web3AuthOptions } from '@web3auth/modal'
+import { OpenloginAdapter } from '@web3auth/openlogin-adapter'
+
+
+// https://web3auth.io/docs/sdk/web/modal/initialize#arguments
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:28]:
```suggestion
// https://web3auth.io/docs/sdk/pnp/web/modal/initialize#arguments
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+    showOnModal: false
+  },
+  [WALLET_ADAPTERS.METAMASK]: {
+    label: 'metamask',
+    showOnDesktop: true,
+    showOnMobile: false
+  }
+}
+
+// https://web3auth.io/docs/sdk/web/modal/whitelabel#whitelabeling-while-modal-initialization
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:57]:
```suggestion
// https://web3auth.io/docs/sdk/pnp/web/modal/whitelabel#whitelabeling-while-modal-initialization
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 22]
```diff
@@ -0,0 +1,196 @@
...
+await web3AuthModalPack.signOut()
+```
+
+Call `getProvider()` to get the Ethereum provider instance.
+
+```typescript
+web3AuthModalPack.getProvider()
+```
+
+We expose two methods for listening to events, `subscribe()` and `unsubscribe()`. In the `Web3AuthModalPack` case, we can listen to all the events listed [here](https://web3auth.io/docs/sdk/web/modal/initialize#subscribing-the-lifecycle-events).
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:116]:
```suggestion
We expose two methods for listening to events, `subscribe()` and `unsubscribe()`. In the `Web3AuthModalPack` case, we can listen to all the events listed [here](https://web3auth.io/docs/sdk/pnp/web/modal/initialize#subscribing-the-lifecycle-events).
```

File: safe-core-sdk/auth-kit/web3auth.md [hunk 23]
```diff
@@ -0,0 +1,196 @@
...
+
+```typescript
+const web3AuthModalPack = new Web3AuthModalPack({
+  txServiceUrl: 'https://safe-transaction-goerli.safe.global'
+})
+```
+
+## Signing transactions using the Web3AuthModalPack and Protocol Kit
+
+The `Web3AuthModalPack` can be combined with the [Protocol Kit](./protocol-kit/) to connect to a Safe using the `provider` and `signer` of the currently authenticated account.
```
Comment (yagopv) [inline safe-core-sdk/auth-kit/web3auth.md:140]:
```suggestion
The `Web3AuthModalPack` can be combined with the [Protocol Kit](../protocol-kit/) to connect to a Safe using the `provider` and `signer` of the currently authenticated account.
```

File: reference/TEMPLATE.md (hunk 17 above)
Comment (germartinez) [reply reference/TEMPLATE.md:37]: There is caveat 1 and caveat 2, so I will leave it as it is
```
- Caveat1: {Description}
- Caveat2: {Description}
```

File: safe-core-sdk/onramp-kit/monerium.md [hunk 24]
```diff
@@ -23,38 +23,37 @@ This guide demonstrates how to use the `MoneriumPack` with the [`OnRampKit`](htt
...
+  environment: 'sandbox' // Use the proper Monerium environment ('sandbox' | 'production')})
+})
+
+await moneriumPack.init({ safeSdk })
 ```
 
-The `safeSdk` is an instance of the [`Safe`](https://github.com/safe-global/safe-core-sdk/blob/main/packages/protocol-kit/src/Safe.ts) class. For more information on how to instantiate the `protocol-kit` refer to the [Protocol Kit Quickstart section](https://docs.safe.global/learn/safe-core/safe-core-account-abstraction-sdk/protocol-kit#quickstart).
+The `safeSdk` is an instance of the [`Safe`](https://github.com/safe-global/safe-core-sdk/blob/main/packages/protocol-kit/src/Safe.ts) class. For more information on how to instantiate the `protocol-kit` refer to the [Protocol Kit Quickstart section](https://docs.safe.global/safe-core-sdk/protocol-kit).
 
 The `MoneriumPack` will use the Safe address configured in the `safeSdk` to link to your Monerium account.
```
Comment (yagopv) [inline safe-core-sdk/onramp-kit/monerium.md:51]:
```suggestion
The `MoneriumPack` will use the Safe Account address configured in the `safeSdk` to link to your Monerium account.
```

File: safe-core-sdk/onramp-kit/monerium.md [hunk 25]
```diff
@@ -79,10 +78,10 @@ Note. If you use the `sandbox` environment, you can test this flow without KYC i
...
 
-Re-initialize the authentication kit using the `init` method and exchange the code, as per Step 1.
+Re-initialize the `MoneriumPack` using the `init` method and exchange the code, as per Step 1.
 
 ```typescript
-const safeMoneriumClient = await onRampKit.open({ authCode: <The querystring code parameter> });
+const safeMoneriumClient = await moneriumPack.open({ authCode: '<The querystring code parameter>' })
 ```
 
 If the code is exchanged without problems, you will be now authenticated with Monerium and your Safe will be linked!. You can start using the `safeMoneriumClient` instance to interact with your Monerium account. This instance is an enhanced version of the Monerium SDK, with some additional Safe features.
```
Comment (yagopv) [inline safe-core-sdk/onramp-kit/monerium.md:87]:
```suggestion
If the code is exchanged successfully, you will be authenticated with Monerium and your Safe will be linked!. You can start using the `safeMoneriumClient` instance to interact with your Monerium account. This instance is an enhanced version of the Monerium SDK, with some additional Safe Account features.
```

File: safe-core-protocol/plugins.md [hunk 26]
```diff
@@ -0,0 +1,24 @@
+# Safe Plugins
+
+Safe Plugins add custom features to Safe contracts. They are smart contracts that implement Safe's functionality while separating plugin logic from Safes core contracts. A basic Safe does not require any plugin. Adding and removing a plugin requires confirmation from the configured threshold number of owners. Events are emitted whenever a plugin is added or removed, and a plugin transaction succeeds or fails.
+
+Safe Plugins can include daily spending allowances, amounts that can be spent without the approval of other owners, recurring transactions plugins, standing orders that are performed on a recurring set date, e.g. paying your rent, and social recovery plugins, which may allow you to recover a Safe if you lose access to owner accounts. These are just a few examples of how to use plugins, but there are many other ways you could use plugins in your Safe.
```
Comment (yagopv) [inline safe-core-protocol/plugins.md:5]:
```suggestion
Safe Plugins can include daily spending allowances, amounts that can be spent without the approval of other owners, recurring transactions plugins, standing orders that are performed on a recurring set date, e.g. paying your rent, and social recovery plugins, which may allow you to recover a Safe if you lose access to owner accounts. These are just a few examples of how you can use plugins in your Safe, but there are many other ways that you could use them.
```

File: safe-core-sdk/safe-apps/README.md [hunk 27]
```diff
@@ -4,9 +4,9 @@ description: What are Safe Apps?
...
 
-With over $90B worth of digital assets held in Safe's alone, it’s essential for the dapp ecosystem to access the untapped market of multi-signature wallets. Safe Apps introduce a completely new way for developers to build their dapps right into a Multisig interface. Turn your Dapp into an HTML iframe component that can be accessed through the Safe. We are providing extensive developer tooling to make it easy to create, test and integrate Safe Apps into the Safe. This includes an unique design system, reusable components and a Safe App SDK that facilitates the Safe App <> Safe communication.
+With over $90B worth of digital assets held in Safes alone, it’s essential for the dapp ecosystem to access the untapped market of multi-signature wallets. Safe Apps introduce a completely new way for developers to build their dapps right into a Multisig interface. Turn your Dapp into an HTML iframe component that can be accessed through the Safe. We are providing extensive developer tooling to make it easy to create, test and integrate Safe Apps into the Safe. This includes an unique design system, reusable components and a Safe App SDK that facilitates the Safe App <> Safe communication.
```
Comment (yagopv) [inline safe-core-sdk/safe-apps/README.md:7]:
```suggestion
With over $90B worth of digital assets held in Safe accounts alone, it’s essential for the dapp ecosystem to access the untapped market of multi-signature wallets. Safe Apps introduce a completely new way for developers to build their dapps right into a Multisig interface. Turn your Dapp into an HTML iframe component that can be accessed through the Safe. We are providing extensive developer tooling to make it easy to create, test and integrate Safe Apps into the Safe. This includes an unique design system, reusable components and a Safe App SDK that facilitates the Safe App <> Safe communication.
```

**Review by dasanra — APPROVED** [review]

**Pull request merged**